arcgis-ios-sdk-samples/Content Display Logic/Controllers/Color Picker/ColorPickerViewController.swift

```diff
@@ -69,6 +73,8 @@ class ColorPickerViewController: UITableViewController {
             saturation = value
         case brightnessSlider:
             brightness = value
+        case alphaSlider:
```

Comment:
The sliders are continue which means that these comparisons are being made many, many times. Could you make a separate action method for each slider to prevent doing a ton of unnecessary comparisons?

Reply:
Sure, that's a good idea.